HISTORY.rst

```diff
@@ -4,6 +4,10 @@ Shotgun Python API Changelog
 
 Here you can see the full list of changes between each Python API release.
 
+v3.1.1 (2019 August 29)
```

Comment:
How does this relate to this particular pull request? I am seeing a bunch of changes in the diff here that seem unrelated to this PR. Do you need to merge with master?